lib/containers/utils.pm

```diff
@@ -24,9 +24,15 @@ use warnings;
 use version_utils;
 use Mojo::Util 'trim';
 
-our @EXPORT = qw(test_seccomp basic_container_tests get_vars can_build_sle_base check_docker_firewall
+our @EXPORT = qw(loadtest_containers test_seccomp basic_container_tests get_vars can_build_sle_base check_docker_firewall
   get_docker_version check_runtime_version container_ip registry_url);
 
+
+sub loadtest_containers {
+    my ($test, %args) = @_;
+    autotest::loadtest("tests/containers/$test.pm", %args);
+}
+
```

Comment:
This duplicates `load_container_tests`.

Reply:
It's a different thing. This is the test loader. It's basically a wrapper to not specify the path every time. Same as for `loadtest_kernel`. This is pure cosmetics. I can use all the time `loadtest('containers/XYZ')` instead of `loadtest_containers('ZYZ')`

Reply:
OK, but then let's move it to `main_container.pm` too as it's used only there.

Reply:
I can do that, but just wanted to keep container related things in container libs. Same as for kernel: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/LTP/utils.pm#L44

Reply:
I have removed this function to avoid confusion.